renderer/index.js

```diff
@@ -286,7 +315,9 @@ function openPlayer (torrent) {
 }
 
 function deleteTorrent (torrent) {
-  torrent.destroy(update)
+  torrent.destroy(function () {
+    saveTorrents() // updates after writing to config
```

Comment:
Good call. We can add some indication to the UI that it's being deleted later, so the button feels responsive.